packages/plugins/Tips/src/components/TipsButton/useTipsAccounts.ts

```diff
@@ -21,7 +22,12 @@ const supportSources: SocialAddressType[] = [
 export function useTipsAccounts(identity: IdentityResolved | undefined, personaPubkey: string | undefined) {
     const { data: socialAccounts = EMPTY_LIST } = useSocialAccountsAll(identity, supportSources)
     const userId = identity?.identifier?.userId
-    const { data: hiddenAddresses } = useHiddenAddressConfigOf(personaPubkey, PluginID.Web3Profile, userId)
+    const { data: hiddenAddresses } = useHiddenAddressConfigOf(
+        personaPubkey,
+        PluginID.Web3Profile,
+        userId,
+        signWithPersona,
```

Comment:
Better if we import `'@masknet/plugin-infra/dom/context'` directly where we need it.

Reply:
`useHiddenAddressConfigOf` is defined in `@masknet/web3-hooks-base` which is not a plugin and should never access plugin context.